### ISSUE TITLE
2022-6-2

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#848]
271Replace 批量归一化 with 批量标准化；done; (2022-4-21)
272Add 框架 after ResNet; done; (2022-4-21)
273Change all title to black body; (2022-4-21)
274Add 第 before 小节; done; (2022-4-22)
275Check 模块; done; (2022-4-22)
276Modify fig 1.2; done; (2022-4-22)
277Delete period in the end of algorithm; done; (2022-4-22)
278Replace 介绍 with 论述; (2022-4-22)
279Replace logits with 输出; (2022-4-22)
